app/views/idv/inherited_proofing/retrieval.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title t('inherited_proofing.headings.retrieval') %>
+
+<%= render PageHeadingComponent.new.with_content(t('inherited_proofing.headings.retrieval')) %>
+
+  <div class='margin-y-4 text-center'>
+    <%= image_tag(
+          asset_url('shield-spinner.gif'),
+          alt: t('inherited_proofing.headings.retrieval'),
```

Comment:
Was this text alternative provided by UX / design artifacts? I notice elsewhere we tend to use an empty text alternative for spinners, so I'm not sure what's intended to be standard practice. If anything, could be worth following-up on.

Reply:
We'll want to have the attribute (usually enforced by automated accessibility tests), but it can be set as empty to at least make it consistent with others:

```suggestion
          alt: '',
```

Reply:
Done